resources/js/Pages/Rooms/CreatePage.tsx

```diff
@@ -14,23 +14,23 @@ const CreatePage: React.FC = () => {
 
 	function handleSubmit(event) {
 		event.preventDefault();
-		post("/login");
+		post("/rooms/create");
 	}
 
 	return (
 		<Panel
 			title="Create new room"
 			footer={
 				<Button
+					handleClick={handleSubmit}
 					className="!w-auto text-white bg-pink-700 hover:bg-pink-800"
-					type="submit"
 					disabled={processing}
 				>
 					Create
 				</Button>
 			}
 		>
-			<form className="space-y-6" onSubmit={handleSubmit}>
+			<form className="space-y-6">
```

Comment:
probably we would like to have onSubmit callback on form, because if you would press enter when field is active, it will not work or wil not submit incorrectly. So we need to move form outside to also wrap button

Reply:
Good point

Reply:
I was thinking we could use `form` button attribute, because currently submit button is outside the form. Not sure if it will work with enter presses, but seeing that it is native functionality I think it is worth a try.